src/Catel.Test/Catel.Test.Shared/IoC/TypeFactoryFacts.cs

```diff
@@ -258,9 +260,9 @@ public void ThrowsCircularDependencyExceptionForInvalidTypeRequestPath()
 
                 var ex = ExceptionTester.CallMethodAndExpectException<CircularDependencyException>(() => typeFactory.CreateInstance<X>());
 
-                Assert.AreEqual(4, ex.TypePath.AllTypes.Length);
+                Assert.AreEqual(3, ex.TypePath.AllTypes.Length);
                 Assert.AreEqual(typeof(X), ex.TypePath.FirstType.Type);
-                Assert.AreEqual(typeof(X), ex.TypePath.LastType.Type);
+                Assert.AreEqual(typeof(X), ex.DuplicatedRequestInfo.Type);
```

Comment:
I think this should be renamed to "DuplicateRequestInfo"? I think it's an actual duplicate, not a duplicated?